src/Stormpath/Cache/RedisCacheManager.php

```diff
@@ -0,0 +1,16 @@
+<?php namespace Stormpath\Cache;
+
+
+class RedisCacheManager implements CacheManager {
+
+    public function __construct($options)
+    {
+        $this->options = $options;
+    }
+
+    public function getCache()
+    {
+
+        return new RedisCache($this->options);
```

Comment:
What about getting the Cache instance from the cache String key (i.e.: from the region)?